toolkit/tools/internal/file/file.go

```diff
@@ -196,6 +196,13 @@ func Write(data string, dst string) (err error) {
 	return
 }
 
+func WriteWithPerm(data string, dst string, perm os.FileMode) (err error) {
+	logger.Log.Debugf("Writing to (%s)", dst)
```

Comment:
Might be also worth to mention the used permission in the log.

Reply:
Fixed.